src/carbonio-files-ui-common/views/components/versioning/Versioning.test.tsx

```diff
@@ -1003,4 +1011,49 @@ describe('Versioning', () => {
 		await user.click(keepVersion);
 		expect(screen.queryByText(/version marked as to be kept forever/i)).not.toBeInTheDocument();
 	});
+
+	it('should render the banner if the upload version fails for over quota', async () => {
```

Comment:
You should place this test near the UploadView, the test is about that component